docs/INFRASTRUCTURE.md

```
@@ -321,4 +358,4 @@ tail -f /var/log/caddy/access.log
 
 ---
 
-*Last updated: 2024-12-09*
+*Last updated: 2026-01-02*
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)